### ISSUE TITLE
Name

### ISSUE THREAD

**Issue by stufraser1** [GFDRR/rdls-spreadsheet-template#14]
@pzwsk @matamadio 
On naming the conversion tool, we have proposed _Risk Data Library Standard metadata toolkit_ or shorter _RDLS_metadata toolkit_ [see rdls-cove #16](https://github.com/GFDRR/rdls-cove/issues/16)

We need a consistent but usable name for the spreadsheet tool too. My initial suggestions: 
- _RDLS_metadata input sheet_ 
- _RDLS_metadataXlsx_ (I like this because it specifies the format to be used and type of data its for) 
- _RDLS_metadataInput_

**Comment by matamadio** [comment]
Complete template: RDLS_md_template.xlsx

Optionally, it can be split for the 4 components as:

Hazard only:   RDLS_md_hzd_template.xlsx
Exposure only:   RDLS_md_exp_template.xlsx
Vulnerability only:   RDLS_md_vln_template.xlsx
Loss only:   RDLS_md_lss_template.xlsx

**Comment by duncandewhurst** [comment]
A few thoughts on naming:

I think we should avoid shortening metadata to 'md' because that is the file extension for Markdown so I would expect an md template to be a Markdown template.

+1 to including something in the name to indicate that it is a spreadsheet, I think that 'input template' is too generic, since there can be other non-spreadsheet methods of input.

I think we should keep template in the name because it helps to communicate that it is a tool for creating/entering data, rather than simply a spreadsheet that contains RDLS metadata.

Given that we might conceivably want to publish the spreadsheet template in alternative formats, e.g. as Google Sheet or as an Open Document Format file, I think 'spreadsheet template' is preferable to 'XLSX template'.

**Comment by stufraser1** [comment]
All good points. Then I propose, and these are unavoidably long but:
Complete template: RDLS_metadataSpreadsheet_template_full
Hazard only: RDLS_metadataSpreadsheet_template_haz
Exposure only: RDLS_metadataSpreadsheet_template_exp
Vulnerability only: RDLS_metadataSpreadsheet_template_vuln
Loss only: RDLS_metadataSpreadsheet_template_loss

**Comment by duncandewhurst** [comment]
Given that we've clarified that the scope of RDLS is restricted to metadata, I think that we can probably drop the metadata part? Unless you think that it's possible that there might be some other (non-metadata) sort of spreadsheet template developed in the future?

**Comment by stufraser1** [comment]
Good point, let's do that

**Comment by matamadio** [comment]
_RDLS_template_full.xlsx_ named with the extension makes clear it is a spreadsheet, it's redundant to add "spreadsheet"

**Comment by stufraser1** [comment]
And if its not xlsx, as Duncan suggests.
Open doc format would be ODS, which would also work that way. Any others where this wouldn't work?

**Comment by matamadio** [comment]
We can consider it as long as these formats retain the full functionality of the spreadsheet.
Note that google docs supports excel format.
Completely anecdotal, but I had nightmerish experiences with open data formats when it came to cross-platform compatibility.

**Comment by stufraser1** [comment]
Should spreadsheet filename be compatible with (or the same as?) the .json template which would be provided for people entering directly as json if they wanted? This differs from RDLS_schema.json, which includes the full schema details. Then we could have:
rdls_template.json
rdls_template.xlsx
rdls_template.<AnyOtherFormatExtension>

and
rdls_template_haz.xlsx
rdls_template_exp.xlsx
rdls_template_vuln.xlsx
rdls_template_loss.xlsx

With the suggestion to users that they append '_projectName' or '_datasetName' to the filename

A nice to have in the spreadsheet tool would be to name the file you're saving.

**Comment by matamadio** [comment]
Yes this makes most sense to me.

**Comment by stufraser1** [comment]
@pzwsk and I spoke to confirm:

template filenames per [above comment](https://github.com/GFDRR/rdls-spreadsheet-template/issues/14#issuecomment-1702411337)

and the main template to be called "RDLS Spreadsheet Template"

**Comment by duncandewhurst** [comment]
I've renamed the templates in https://github.com/GFDRR/rdls-spreadsheet-template/commit/d7a41f6a698d63ec0e612fa4161fa13c71c9020d, it's a little complicated to abbreviate the different components to different lengths so I just used the full component names. Hope that's OK.

> A nice to have in the spreadsheet tool would be to name the file you're saving.

Users can do File > Save As to name the completed template. Specifying the name of the converted JSON file would need to be implemented in CoVE - feel free to open an issue on the rdls-cove repo if that would be a useful feature.
